src/webots/vrml/WbParser.hpp

```diff
@@ -63,7 +63,7 @@ class WbParser {
   WbTokenizer *mTokenizer;
   int mMode;
 
-  enum { NONE, WBT, VRML, PROTO, WBO, WRL };
+  enum { NONE, WBT, VRML, PROTO, WRL };
```

Comment:
Is there a reason to keep WRL here, as it was removed at multiple other locations in the code?

Reply:
They were used to parse the [resources/nodes/*.wrl](https://github.com/cyberbotics/webots/tree/master/resources/nodes) files. However, I found a more elegant way to handle this. See 610b178d899f314fc274d9c2282cb7e25e74a9a4